src/app/shop/[category]/[productId]/_components/TabContents/Review/KeywordRatios.tsx

```diff
@@ -0,0 +1,48 @@
+import { REVIEW_KEYWORD } from '@/constants/reviewKeyword';
+import type { KeywordStatistics, OptionRatio } from '@/types/ProductReviewTypes';
+import classNames from 'classnames/bind';
+import styles from './KeywordRatio.module.scss';
+
+const cn = classNames.bind(styles);
+
+interface KeywordRatiosProps {
+  optionRatios: KeywordStatistics;
+}
+
+export default function KeywordRatios({ optionRatios }: KeywordRatiosProps) {
+  const { option1Ratios, option2Ratios, option3Ratios } = optionRatios;
+
+  const category = '키보드';
```

Comment:
```suggestion
  const category = '키보드';
```
상수는 스네이크케이스로 컴포넌트 밖으로 꺼내쓰는게 저희 컨벤션에 더 맞는거같아요

Reply:
어머 예시로 적어놓은건데 pathname 따와서 동작하게 수정했습니다 !